### PR TITLE
[GH-4699]: Improve share dialog for nested docs

### LINKED ISSUE #4699: Share dialog for a nested shared document improves
In this dialog that is shown on a nested document within a publicly shared document we should have some way of getting to the top-level share. It warns you that a parent is publicly shared but doesn't provide a way to copy a link from that share.

image

### PR DESCRIPTION
**Pull request by adityash1**
Fixes #4699

### CODE REVIEW

File: app/scenes/Document/components/SharePopover.tsx [hunk 1]
```diff
@@ -226,15 +227,15 @@ function SharePopover({
...
-            <Trans
-              defaults="This document is shared because the parent <em>{{ documentTitle }}</em> is publicly shared"
-              values={{
-                documentTitle: sharedParent.documentTitle,
-              }}
-              components={{
-                em: <strong />,
-              }}
-            />
+            {t("This document is shared because the parent ")}
```
[the remark this answers is not in the document]
Comment (apoorv-mishra) [reply app/scenes/Document/components/SharePopover.tsx:230]: What's failing exactly? Error log or something similar would be more suggestive.

[the remark this answers is not in the document]
Comment (adityash1) [reply app/scenes/Document/components/SharePopover.tsx:230]: I mean, how can we make a particular text hyperlink using react router's `Link` inside `Trans` component.

[the remark this answers is not in the document]
Comment (tommoor) [reply app/scenes/Document/components/SharePopover.tsx:230]: It is possible to use the other style where `children` are passed to `Trans`, we just don't do it very often. See docs: 

https://react.i18next.com/latest/trans-component

[the remark this answers is not in the document]
Comment (adityash1) [reply app/scenes/Document/components/SharePopover.tsx:230]: Thanks. I should've read it before asking…

**Comment by adityash1** [comment]
@tommoor this lgtm. Let me know if needed further changes

**Review by tommoor — APPROVED** [review]
Thanks

**Pull request merged**